src/components/AboutUs.js

```diff
@@ -55,17 +63,15 @@ const AboutUs = () => (
         </MobileGameTableWrapper>
 
         <Description>
-          State of the art tech and design which make your customer interface delightful and unbreakable
+          State of the art <Em>tech</Em> and <Em>design</Em> for <Em>delightful</Em> and <Em>unbreakable</Em> interfaces
```

Comment:
I don't know if we'll be able to use a styled component inside a FormattedMessage

Reply:
hmm yeah maybe you'll need to split over them.

Reply:
would not better do exactly what we have below this component?